internal/app/app.go

```diff
@@ -1467,6 +1475,10 @@ func (app *App) repairSlaveOfflineMode(host string, node *mysql.Node, state *Nod
 			} else {
 				app.logger.Infof("repair: slave %s set offline, because replication permanently broken", host)
 			}
+			err = node.OptimizeReplication()
+			if err != nil {
+				app.logger.Errorf("repair: failed to set optimize replication settings on slave %s: %s", host, err)
+			}
```

Comment:
`replPermBroken` is `true`. Does it make sense to change replication settings in this case?